app/globals.css

```diff
@@ -179,6 +179,10 @@
     --delay-step-increment: 0.25s;
     --animation-length: 3s;
     animation: wave-y-5 var(--animation-length) ease-in-out infinite;
```

Comment:
Turn the logo animation into a gentler logo animation if the user prefers-reduced-motion. Keeping a little animation allows us to keep the pause/play icon, too